File: 소프티어/DCT.py
import sys

input = sys.stdin.readline

# 1단 ~ 8단 : ascending
# 8단 ~ 1단 : descending
# else: mixed
# 1부터 8까지 숫자는 한번씩 등장

DCT = list(map(int, input().split()))
ASC = sorted(DCT)
DES = sorted(DCT, reverse=True)

print('ascending' if DCT==ASC else 'descending' if DCT==DES else 'mixed')

File: 소프티어/DCT.js
var dct = 0
for (var i = 0; i < input.length - 1; i++) {
    if (Number(input[i + 1]) == Number(input[i]) + 1) {
        dct = 1;
    } else if (Number(input[i]) - 1 == Number(input[i + 1])) {
        dct = -1;
    } else {
        dct = 0;
        // 중간에 어긋나고 다시 순서대로 가는 것 방지
        break;
    }
}

dct == 1 ? console.log('ascending') : dct == -1 ? console.log('descending') : console.log('mixed');